distributed/tests/test_joblib.py

```diff
@@ -32,14 +32,22 @@ def test_simple(loop):
             seq = Parallel()(delayed(inc)(i) for i in range(10))
             assert seq == [inc(i) for i in range(10)]
 
+            from joblib.parallel import get_active_backend
+            ba, _ = get_active_backend()
+            ba.executor.shutdown()
```

Comment:
@ogrisel it would be nice for the backends to have the ability to clean themselves up